kpconv_torch/datasets/Toronto3D.py

```diff
@@ -15,31 +15,36 @@
 from kpconv_torch.utils.mayavi_visu import show_input_batch
 from kpconv_torch.utils.ply import read_ply, write_ply
 from kpconv_torch.utils.tester import get_test_save_path
-from kpconv_torch.utils.trainer import get_train_save_path
 
 
 class Toronto3DDataset(PointCloudDataset):
     """Class to handle Toronto3D dataset."""
 
     def __init__(
         self,
-        config,
         command,
+        config,
         datapath,
         chosen_log=None,
         infered_file=None,
         output_dir=None,
-        split="training",
         use_potentials=True,
         load_data=True,
+        split="training",
     ):
+        PointCloudDataset.__init__(
+            self,
+            config=config,
+            datapath=datapath,
+            dataset="Toronto3D",
+            chosen_log=chosen_log,
+            infered_file=infered_file,
+            output_dir=output_dir,
+            split=split,
+        )
         """
         This dataset is small enough to be stored in-memory, so load all point clouds here
         """
-        PointCloudDataset.__init__(self, "Toronto3D")
-
-        self.train_save_path = get_train_save_path(output_dir, chosen_log)
-        self.test_save_path = get_test_save_path(infered_file, chosen_log)
```

Comment:
Same remarks:

- do not add code before docstring
- use `super().__init__(...)` to call the constructor of the parent class